app/src/main/java/org/simple/clinic/home/patients/PatientsInit.kt

```diff
@@ -14,7 +14,8 @@ class PatientsInit(
         LoadUser,
         RefreshUserDetails,
         LoadNumberOfPatientsRegistered,
-        LoadInfoForShowingAppUpdateMessage
+        LoadInfoForShowingAppUpdateMessage,
+        LoadAppStaleness
```

Comment:
I think trying to load app staleness here is causing the tests to fail? 

I am just curious, we are already passing priority in `ShowAppUpdate`, any reason why we didn't do this for staleness as well? Is there any issue with that? 🤔

Reply:
Not a bad idea. Will do this in the next pr